db/db_impl/db_impl_write.cc

```diff
@@ -263,16 +263,19 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
     size_t total_count = 0;
     size_t valid_batches = 0;
     size_t total_byte_size = 0;
+    size_t pre_release_callback_cnt = 0;
```

Comment:
The changes to db_impl_write.cc are separately approved in https://github.com/facebook/rocksdb/pull/5381